### ISSUE TITLE
Dkron doesn't work like a service on Ubuntu 16.04

### ISSUE THREAD

**Issue by giovannialbero1992** [victorcoder/dkron#230]
I've installed dkron by apt repository and by .deb file but when i write 
```sudo service dkron start``` 
I can see this response
```Failed to start dkron.service: Unit dkron.service not found.```

Can be a problem with Ubuntu's version?

Thanks

**Comment by giovannialbero1992** [comment]
@victorcoder I've added 
```
[Unit]
Description=Dkron Agent

[Service]
User=root
ExecStart=/usr/bin/dkron agent -ui-dir=/usr/share/dkron >> /var/log/dkron.log 2>&1

[Install]
WantedBy=multi-user.target
```
into `/etc/systemd/dkron.service` (I've created this file previous) and seems to work.

**Comment by victorcoder** [comment]
Hi @giovannialbero1992, yes this was a pending issue, thanks for sharing, I'll add the service to the package.

**Comment by shaun-ba** [comment]
@victorcoder where is the dkron.service located? i'm just getting this:

sudo systemctl status dkron
● dkron.service
   Loaded: not-found (Reason: No such file or directory)
   Active: inactive (dead)

**Comment by victorcoder** [comment]
@shaun-ba it's not in the stable branch yet, It'll be released in the next version, 0.9.3 in the following days.
